pages/index.js

```diff
@@ -53,15 +55,18 @@ export function getServerSideProps(context) {
   }
 
   return {
-    props: {}
+    props: {},
+    redirect: {
+      destination: config.WIX_URL,
+    },
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The redirect to `config.WIX_URL` is a significant change. It's important to ensure that the URL stored in this configuration variable is valid and secure. If it's user-provided data, it should be sanitized to prevent any potential security issues such as open redirects.



<!-- This is an auto-generated comment by OSS CodeRabbit -->